Lib/test/datetimetester.py

```diff
@@ -6223,7 +6223,7 @@ def nondst_folds(self):
         for (_, prev_ti), (t, ti) in pairs(zip(self.ut, self.ti)):
             shift = ti[0] - prev_ti[0]
             if shift < ZERO and ti[1] == prev_ti[1]:
-                yield _utcfromtimestamp(datetime, t,), -shift, prev_ti[2], ti[2]
+                yield datetime.fromtimestamp(t, datetime.UTC), -shift, prev_ti[2], ti[2]
```

Comment:
```suggestion
                yield datetime.utcfromtimestamp(t), -shift, prev_ti[2], ti[2]
```

[Discussion](https://discuss.python.org/t/where-is-utcfromtimestamp-defined/88612)

Reply:
`utcfromtimestamp` is deprecated.